modules/services/elasticsearch/variables.tf

```diff
@@ -10,3 +10,26 @@ variable "number_of_nodes" {
 variable "create_iam_service_linked_role" {
   default = true
 }
+
+variable "zone_awareness_enabled" {
```

Comment:
Please add these new input variables to [elasticsearch/README](https://github.com/open-craft/terraform-scripts/tree/635890d9fb48661c6f531df494b1042bd23e5f1d/modules/services/elasticsearch).

(Might be able to remove `number_of_subnets` and `start_number_of_subnets` depending on what you think about [this comment](https://github.com/open-craft/terraform-scripts/pull/18/files#diff-0f9cb3263848eba15ae3facdaa4b00ff2365de9c827c879aa4678d49855f6688R31)).

Reply:
added :+1: